BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -743,6 +744,26 @@ public async Task<IActionResult> SearchForHearingsAsync([FromQuery] SearchForHea
             var response = hearingMapper.MapHearingToDetailedResponse(hearings, caseNumber);
             return Ok(response);
         }
+        
+        /// <summary>
+        /// Get all the unallocated hearings
+        /// </summary>
+        /// <returns>unallocated hearings</returns>
+        [HttpGet("unallocated")]
+        [OpenApiOperation("GetUnallocatedHearings")]
+        [ProducesResponseType(typeof(List<HearingDetailsResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        public async Task<IActionResult> GetUnallocatedHearings()
+        {
+
+            var results = await _hearingService.GetUnallocatedHearings();
+
+            if (results.Count <= 0)
```

Comment:
think we should return an empty list here rather than a 404, otherwise the scheduler job will fail when there are no hearings to allocate to

Reply:
refactor to return empty list